server/feeders/reservationsgenerator.js

```diff
@@ -56,29 +55,38 @@ var feedOrionReservations = function() {
   };
 
   Object.keys(restaurantsData).forEach(function(element, pos) {
```

Comment:
I think it would be better to name the second argument as 'index' instead of 'pos'